webplugin/js/app/km-richtext-markup-1.0.js

```diff
@@ -696,5 +715,18 @@ Kommunicate.markup.getGenericButtonMarkup = function (metadata) {
     return buttonContainerHtml + "</div>";
 
 }
+Kommunicate.markup.getVideoMarkup = function (options) {
+    var video;
+    if (options && options.payload) {
+        var payload = typeof options.payload == 'string' ? JSON.parse(options.payload) : {};
+        for (var i = 0; i < payload.length; i++) {
+            video = payload[i];
```

Comment:
'var video' should be inside for loop

Reply:
corrected, 'video' variable declared inside the for loop